pandas/tests/io/formats/test_to_latex.py

```diff
@@ -972,6 +972,30 @@ def test_to_latex_float_format_no_fixed_width_integer(self):
         )
         assert result == expected
 
+    @pytest.mark.parametrize("na_rep", ["NaN", "Ted"])
+    def test_to_latex_na_rep_and_float_format(self, na_rep):
+        df = DataFrame(
+            [
+                ["A", 1.2225],
+                ["A", None],
+            ],
+            columns=["Group", "Data"],
+        )
+        result = df.to_latex(na_rep=na_rep, float_format="{:.2f}".format)
+        expected = _dedent(
```

Comment:
Nice. I think this same pattern appears in other places, could push this definition higher up and use elsewhere?

Reply:
Do you mean ``_dedent`` (without the preceding newline?)

Reply:
Ah, maybe they aren't using the preceding newline so could probably use dedent alone for those

Reply:
Here I used custom ``_dedent`` only to make it compatible with raw strings.
In general I would simply use this
```
dedent(
    """\
    content
    """
)
```